src/coffee/run.coffee

```diff
@@ -258,10 +257,18 @@ ensureCredentials(argv)
                 if exportType.toLowerCase() is 'csv' and argv.createSyncActions
                   createSyncOrders(@orderReferences[0].fileName)
                     .then (orders) =>
-                      orderJsonFile = "tobeSyncOrders.json" # Can be renamed??
-                      fs.writeFileAsync("#{@outputDir}/#{orderJsonFile}", JSON.stringify(orders,null,2)).then =>
-                        logger.debug "Uploading #{@outputDir}/#{orderJsonFile}"
-                        sftpClient.safePutFile(sftp, "#{@outputDir}/#{orderJsonFile}", "#{sftpTarget}/#{orderJsonFile}")
+                      if orders.length
+                        if argv.fileWithTimestamp
```

Comment:
I was following the already set standards for the tool, but yeah settings are harmful 👿👺